lib/buoys/loader.rb

```diff
@@ -19,17 +19,31 @@ def buoys
         @buoys ||= {}
       end
 
+      def loaded_times
+        @loaded_times ||= []
+      end
+
       def load_buoys_files
+        return unless need_reload?
+
         buoys.clear
+        loaded_times.clear
 
         buoy_files.each do |file|
           instance_eval open(file).read, file
+          loaded_times << File.mtime(file)
         end
       end
 
       def buoy_files
         Dir[*Buoys.buoy_file_paths]
       end
+
+      def need_reload?
+        return true if buoys.empty?
+
+        Rails.env.development? && loaded_times != buoy_files.map {|f| File.mtime(f) }
```

Comment:
I don't want to reload it as much as possible, even if in development. Maybe, every time all reloading buoys is slow.